app/helpers/audits_helper.rb

```diff
@@ -42,6 +42,11 @@ def details(audit)
           (_("Provisioning Template content changed %s") % (link_to 'view diff', audit_path(audit))).html_safe if audit_template? audit
         elsif name == "owner_id" || name == "owner_type"
           _("Owner changed to %s") % (audit.revision.owner rescue _('N/A'))
+        elsif name == 'global_status'
+          base = audit.audited_changes.values[0]
+          from = HostStatus::Global.new(base[0]).to_label
+          to = HostStatus::Global.new(base[1]).to_label
+          _("Global status changed from %{from} to %{to}") % { :from => from, :to => to }
```

Comment:
I don't think so, API returns easy to parse data. In other audits we display null instead of empty string and true/false instead of "pending build"/"built".